constants/chatbotCommands.py

```diff
@@ -2497,6 +2497,16 @@ async def mpScoreV() -> Optional[str]:
     async def mpHelp() -> Optional[str]:
         return f"Supported multiplayer subcommands: <{' | '.join(subcommands.keys())}>."
 
+    async def mp_link() -> Optional[str]:
+        multiplayer_match = await matchList.getMatchFromChannel(chan)
+        assert multiplayer_match is not None
+
+        message = f"Match history available [https://akatsuki.gg/matches/{multiplayer_match['match_id']} here]."
+        if multiplayer_match["match_history_private"]:
+            message += " This is only visible to participants of this match!"
```

Comment:
might be nice to move this somewhere else since it's also used on match join

Reply:
resolved in 67548613ea8fb6da0adaec96f7797fc2b8800ef4